modules/node/persistence/cms_node_storage_sql.e

```diff
@@ -260,9 +263,9 @@ feature -- Helpers
 
 feature {NONE} -- Queries
 
-	sql_select_nodes_count: STRING = "SELECT count(*) from Nodes;"
+	sql_select_nodes_count: STRING = "SELECT count(*) from Nodes where deleted_at IS NULL;"
```

Comment:
please always put SQL keyword as uppercase, this is then much easier to quickly read a SQL statement.